cmd/libs/go2idl/client-gen/main.go

```diff
@@ -48,6 +48,7 @@ var (
 	includedTypesOverrides = flag.StringSlice("included-types-overrides", []string{}, "list of group/version/type for which client should be generated. By default, client is generated for all types which have genclient=true in types.go. This overrides that. For each groupVersion in this list, only the types mentioned here will be included. The default check of genclient=true will be used for other group versions.")
 	basePath               = flag.String("input-base", "k8s.io/kubernetes/pkg/apis", "base path to look for the api group. Default to \"k8s.io/kubernetes/pkg/apis\"")
 	clientsetName          = flag.StringP("clientset-name", "n", "internalclientset", "the name of the generated clientset package.")
+	clientsetAPIPath       = flag.StringP("clientset-api-path", "", "", "the value of default API path.")
```

Comment:
I think defaulting to "" is better. We need to distinguish if "/apis" is user specified or if it's the default. It matters because we need to treat the "core" group specially in the latter case.